h2o-py/tests/testdir_algos/glm/pyunit_glm_makeGLMModel_PUBDEV_5442.py

```diff
@@ -0,0 +1,31 @@
+import sys
+sys.path.insert(1,"../../../")
+import h2o
+from builtins import range
+from tests import pyunit_utils
+from h2o.estimators.glm import H2OGeneralizedLinearEstimator as glm
+
+def test_makeGLMModel():
+    # read in the dataset and construct training set (and validation set)
+    d = h2o.import_file(path=pyunit_utils.locate("smalldata/logreg/prostate.csv"))
+    m = glm(family='binomial',Lambda=[0.001], alpha=[0.5],solver='COORDINATE_DESCENT')
+    m.train(training_frame=d,x=[2,3,4,5,6,7,8],y=1)
+    r = glm.getGLMRegularizationPath(m)
+    m2 = glm.makeGLMModel(model=m,coefs=r['coefficients'][0])
+    f1 = m.predict(d)   # predict with original model
+    f2 = m2.predict(d)  # predict with model out of makeGLMModel
+    pyunit_utils.compare_frames_local(f1[1],f2[1],prob=1)
+    coefs = r['coefficients'][0]
+    coefs['wendy_dreams']=8
+    
+    try:
+        mw = glm.makeGLMModel(model=m,coefs=coefs)
+        sys.exit("Should have throw exception of bad coefficient length")
+    except Exception as ex:
+        print(ex)
```

Comment:
should assert on execption message at least

Reply:
Done.  Thanks!